library/Fission/Web/IPFS.hs

```diff
@@ -41,11 +39,6 @@ type UnauthedAPI = "cids" :> CID.API
               :<|> Pin.API
               :<|> "dag" :> DAG.API
 
-type SimpleAPI = "cids" :> CID.API
-            :<|> Upload.Simple.API
-            :<|> Pin.API
-            :<|> "dag" :> DAG.API
```

Comment:
Moved to the only module that cares about it

Reply:
Newbie question:
Why is this true:
> keeps this module more server-focused

It seems like this file is fairly server focused?